config/apache-download.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+    echo "Argument must be file path to apache download. (e.g. hadoop/common/hadoop-3.1.1/hadoop-3.1.1.tar.gz )"
+    exit 1
+fi
+
+FILE_PATH=$1
+REDIRECT_BASE_URL="http://apache.rediris.es/"
```

Comment:
Given that our hosting is mostly in Germany and not spain, should we use a mirror in Germany? (.de)